Makefile

```diff
@@ -216,6 +216,11 @@ migrate: ## perform database migrations
 	@$(MANAGE) migrate
 .PHONY: migrate
 
+makemigrations: ## create database migrations
```

Comment:
about your commit messages, it would be better to avoid merge commits. You might want to read our contribution guidelines out git messages: https://handbook.openfun.fr/git